cli.go

```diff
@@ -432,6 +432,111 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:  "follow-massively",
```

Comment:
Let's be more specific.

```suggestion
				Name:  "follow-by-lang",
```